### DISCUSSION TITLE
Process is not defined

### DISCUSSION THREAD

**Discussion by sn06work** [webpack/webpack#17211] [Q&A]
dex.js:23128 Uncaught ReferenceError: process is not defined
    at eval (util.js:109:1)
    at ./node_modules/util/util.js (index.js:20916:1)
    at __webpack_require__ (index.js:23125:33)
    at fn (index.js:23402:21)
 Can anyone help me with this error this is coming with webpack 5 version when I try to run wdio test case

**Comment by alexander-akait** [comment]
It means you have `process` in your dependecies, do you have `target: "web"`? If you please read https://stackoverflow.com/questions/65018431/webpack-5-uncaught-referenceerror-process-is-not-defined, there is a good solution

**Reply by sn06work** [reply]
@alexander-akait I don't have process in the dependencies.
This is the full context of error: 
<img width="889" alt="image" src="https://github.com/webpack/webpack/assets/128142423/b77434f3-fa24-4fb0-8cef-b07978854355">
I do went through the above attached link and webpack-dev-server  version 4 upgarde guide

**Reply by alexander-akait** [reply]
Disable `devtool: false` and look at your code
